inc/task.view.class.php

```diff
@@ -522,6 +522,44 @@ function showForm($id, $options=array()) {
       return true;
    }
 
+   function showFormButtons($options = array()) {
+      if (isset($this->fields['id'])) {
+         $ID = $this->fields['id'];
+      } else {
+         $ID = 1;
```

Comment:
Hum, it's a copy paste from https://github.com/glpi-project/glpi/blob/master/inc/commondbtm.class.php#L2092
I think for the case of fi tasks, we can clean this else